pages.fr/common/atoum.md

```diff
@@ -0,0 +1,36 @@
+# atoum
+
+> Un framework de test unitaire pour PHP simple, moderne et intuitif.
+> Plus d'informations : <http://atoum.org>.
+
+- Initialise un fichier de configuration :
+
+`atoum --init`
+
+- Lance les tests:
```

Comment:
```suggestion
- Lance les tests :
```

Reply:
i think i should add a rule on the the linter of tldr for this case 🤔 

wdyt ?